### ISSUE TITLE
혹시 챗봇 API에서 커스텀챗봇 에디터 시나리오를 끌고올수있나요

### ISSUE THREAD

**Issue by 5wakeup-dev** [navertalk/chatbot-api#201]
작성한 질문-답변 내역을 끌고오거나 검색하거나 하는 기능이 제공되고 있을까요?

**Comment by navertalk** [comment]
안녕하세요.

커스텀 챗봇의 시나리오를 가져오거나 불러오는 기능은 제공되고 있지 않습니다.

감사합니다.

p.s. 커스텀 챗봇을 사용하시다가 챗봇 API로 전환하는 작업을 진행중이신가요?

**Comment by 5wakeup-dev** [comment]
> 

예 정확히는 커스텀 챗봇에서 자연어 검색이 가능할까 하여 찾아보고 있었는데 정보가 안나와서 API쪽을 찾아보고있었습니다.
커스텀 챗봇에 등록해놓은 시나리오 양이 많아서 가져올수있을까 해서요
혹시 커스텀 챗봇에서 버튼이 아닌 대화입력으로 원하는 답변이 나오게 하는 기능이 있을까요?

**Comment by navertalk** [comment]
안녕하세요.

커스텀 챗봇이 사용자의 텍스트 입력에 따라 자연어 처리를 통해 시나리오를 분기하는 흐름을 말씀하시는 것 같습니다.
아쉽지만 이런 기능은 제공하고 있지 않습니다. 😭

감사합니다.

**Comment by 5wakeup-dev** [comment]
넵 답변 감사합니다!
